tests/test_attention_embeddings.py

```diff
@@ -85,3 +81,24 @@ def test_parallel_processing(model_name):
         assert np.allclose(emb_1.indices, emb_3.indices)
         assert np.allclose(emb_1.values, emb_2.values)
         assert np.allclose(emb_1.values, emb_3.values)
+
+
+@pytest.mark.parametrize("model_name", ["Qdrant/bm25"])
+def test_multilanguage(model_name):
+    docs = ["Mangez-vous vraiment des grenouilles?", "Je suis au lit"]
+
+    model = SparseTextEmbedding(model_name=model_name, language="french")
+    embeddings = list(model.embed(docs, parallel=2))[:2]
```

Comment:
```suggestion
    embeddings = list(model.embed(docs))[:2]
```

Reply:
we've already tested parallel above, i think it's better to keep this test simpler and check only one thing